tests/pytype_test.py

```diff
@@ -83,34 +106,56 @@ def communicate(self):
         return self.results
 
 
+def _get_relative(filename):
+    top = filename.find('stdlib')
```

Comment:
adding '/' after 'stdlib' might make this a tiny bit more robust.

Reply:
done